dev/devicelab/bin/tasks/flutter_engine_group_performance.dart

```diff
@@ -20,6 +20,8 @@ Future<void> _withApkInstall(
   final DeviceDiscovery devices = DeviceDiscovery();
   final AndroidDevice device = await devices.workingDevice as AndroidDevice;
   await device.unlock();
+  // Force proper cleanup before trying to install app.
+  await device.adb(<String>['uninstall', bundleName]);
```

Comment:
If this command fails does the test fail?

Reply:
Yes, it will fail...so I think we need to first check if the app is installed before uninstalling it because it's not clear to me that `uninstall` will not fail if the app is not installed.

Reply:
Or figure out how to make this command not failing.

Reply:
I am assuming there is an argument we can set if not then I think `| true` after the install will prevent the code from failing in the terminal.